atc/exec/across_step_test.go

```diff
@@ -304,7 +374,7 @@ var _ = Describe("AcrossStep", func() {
 				Expect(ok).To(BeFalse())
 
 				By("ensuring not all steps were started")
-				Expect(started).ToNot(HaveLen(12))
+				Expect(started).ToNot(HaveLen(24))
 			})
```

Comment:
This test will hang forever. Because new elements were added to the `allVals` [array above](https://github.com/concourse/concourse/pull/8271/files#diff-48f60c9a9abecfb45a8c06695920f378782827b0c4bc81dc3a92c66e17d6c446R238-R241), this test case is now trying to terminate the `{"a1", "b1", "c1", "d2"}` step, but this step is not part of the first stage and so the the steps there will never terminate.

This can be fixed by changing line 369 to point to one of the vars combo that is executed in the first stage.